test/verify/machineslib.py

```diff
@@ -1264,6 +1264,20 @@ def installWithErrorTest(dialog):
                                              os_name=config.NOVELL_NETWARE_6,
                                              start_vm=False))
 
+            self.machine.execute("qemu-img create -f qcow2 /var/lib/libvirt/vmTest20.qcow2 128M")
+            self.machine.execute("virsh pool-refresh libvirt")
+
+            # Check choosing existing volume as destination storage
+            createTest(TestMachines.VmDialog(self, "subVmTestCreate20", sourceType='pxe',
```

Comment:
Please don't use PXE. The test is very slow, just choose another installation method.